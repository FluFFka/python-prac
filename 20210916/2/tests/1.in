[2, 5, 1, 10100010101010, 2349239293, 2, 2, 3]
