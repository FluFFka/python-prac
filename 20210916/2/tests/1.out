[1, 2, 2, 2, 3, 5, 2349239293, 10100010101010]
